pandas/core/generic.py

```diff
@@ -457,12 +457,39 @@ def axes(self):
 
     @property
     def ndim(self):
-        """Number of axes / array dimensions"""
+        """
+        Return an int representing the number of axes / array dimensions.
+
```

Comment:
I think it would be fine to add here a sentence saying that this is always 1 for Series and 2 for DataFrame

Reply:
Done.